kernel_hardening_checker/checks.py

```diff
@@ -652,12 +652,24 @@ def add_cmdline_checks(l: List[ChecklistObjType], arch: str) -> None:
     l += [OR(CmdlineCheck('self_protection', 'kspp', 'kfence.sample_interval', '100'),
              AND(KconfigCheck('self_protection', 'kspp', 'KFENCE_SAMPLE_INTERVAL', '100'),
                  CmdlineCheck('-', '-', 'kfence.sample_interval', 'is not set')))]
+    l += [OR(CmdlineCheck('self_protection', 'kspp', 'lockdown', 'confidentiality'),
+             AND(KconfigCheck('self_protection', 'kspp', 'LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY', 'y'),
+                 CmdlineCheck('-', '-', 'lockdown', 'is not set')))]
+             # consequence of the LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY check by kspp
+    l += [OR(CmdlineCheck('self_protection', 'kspp', 'module.sig_enforce', '1'),
+             KconfigCheck('cut_attack_surface', 'kspp', 'MODULES', 'is not set'),
+             AND(KconfigCheck('self_protection', 'kspp', 'MODULE_SIG_FORCE', 'y'),
+                 CmdlineCheck('-', '-', 'module.sig_enforce', 'is not set')))]
+             # consequence of the MODULE_SIG_FORCE check by kspp
     if arch in ('X86_64', 'ARM64', 'X86_32', 'RISCV'):
         l += [OR(CmdlineCheck('self_protection', 'kspp', 'randomize_kstack_offset', '1'),
                  AND(KconfigCheck('self_protection', 'kspp', 'RANDOMIZE_KSTACK_OFFSET_DEFAULT', 'y'),
                      CmdlineCheck('-', '-', 'randomize_kstack_offset', 'is not set')))]
     if arch in ('X86_64', 'X86_32'):
         l += [CmdlineCheck('self_protection', 'kspp', 'mitigations', 'auto,nosmt')]
+        l += [OR(CmdlineCheck('self_protection', 'kspp', 'intel_iommu', 'on'), # consequence of 'INTEL_IOMMU_DEFAULT_ON' kconfig check by kspp
```

Comment:
Same with `intel_iommu`. We can't require it to be `on` or not set. Any ideas?

Reply:
I double-checked the variants of `intel_iommu`. Looks like we can require `on` by default.

If some users explicitly override this value, then they know what they are doing.

Please also skip normalization for `intel_iommu`.

Reply:
After your fixes, could you please squash this branch into a single commit and rebase onto the current `master`?

Thanks!